site/content/communities/bay-area/twitter.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Twitter - Bay Area
+description: Keep posted on the latest news and events impacting Latinx in the Tech industry in the Bay Area and beyond.
```

Comment:
change the desc to something more specific to our group

Reply:
Yeah, this can be solved in a future PR as I'm not sure what the copy should be. 👍